WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -350,13 +354,12 @@ + (NSArray *)routes
   return FBResponseWithOK();
 }
 
-+ (XCUICoordinate *)getGestureCoordinate:(FBRouteRequest *)request
++ (XCUICoordinate *)gestureCoordinateWithCoordinate:(CGPoint)coordinate application:(XCUIApplication *)application
 {
-  FBSession *session = request.session;
-  CGVector point = CGVectorMake((CGFloat)[request.arguments[@"x"] doubleValue], (CGFloat)[request.arguments[@"y"] doubleValue]);
-  XCUICoordinate *appCoordinate = [[XCUICoordinate alloc] initWithElement:session.application normalizedOffset:CGVectorMake(0, 0)];
-  XCUICoordinate *gestureCoordinate = [[XCUICoordinate alloc] initWithCoordinate:appCoordinate pointsOffset:point];
-  return gestureCoordinate;
+  // TODO: handle this properly after XCTest starts to respect landscape interface orientation
+  CGPoint point = FBInvertPointForApplication(coordinate, application.frame.size, application.interfaceOrientation);
```

Comment:
> You should check for iOS version here as well. 
This might be a bad idea, because you never know whether  given coordinates were calculated based on some element position or just based on the actual screen size. I would prefer to keep it like this, because this is generic method and only keep this verification for methods, where we explicitly calculating offset based on element location (like tap)